appinventor/blocklyeditor/src/blocks/lists.js

```diff
@@ -468,3 +467,1519 @@ Blockly.Blocks['lists_join_with_separator'] = {
   },
   typeblock: [{ translatedName: Blockly.Msg.LANG_LISTS_JOIN_WITH_SEPARATOR_TITLE }]
 };
+
+Blockly.Blocks['lists_mutatorcontainer'] = {
+  init: function() {
+    this.setColour(Blockly.LIST_CATEGORY_HUE);
+    var group = new Blockly.RadioButtonGroup();
+    this.appendDummyInput()
+        .appendField(new Blockly.FieldRadioButton(group), 'CHANGE_LIST')
+        .appendField("changes existing list");
+    this.appendDummyInput()
+        .appendField(new Blockly.FieldRadioButton(group), 'MAKE_NEW_LIST')
+        .appendField("makes new list");
+    this.contextMenu = false;
+  }
+};
+
+Blockly.Blocks['lists_map'] = {
+  // For each loop.
+  category : 'Lists',
+  helpUrl : Blockly.Msg.LANG_LISTS_MAP_HELPURL,
+  init : function() {
+    this.setColour(Blockly.LIST_CATEGORY_HUE);
+    this.appendValueInput('LIST')
+       .setCheck(Blockly.Blocks.Utilities.YailTypeToBlocklyType("list",Blockly.Blocks.Utilities.INPUT))
```

Comment:
Our convention is to indent these chained method calls by 2 spaces.

There's a bunch of these below in this file, please re-indent them all.

Reply:
Fixed